drivers/storage/ebs/tests/ebs_test.go

```diff
@@ -138,7 +174,7 @@ func TestInstanceIDEC2(t *testing.T) {
 	}
 
 	// create storage driver
-	sd, err := registry.NewStorageDriver("ec2")
+	sd, err := registry.NewStorageDriver("ebs")
```

Comment:
Is this change necessary? I believe this part makes sure that things still work if someone is using `ec2` in their config file isntead of `ebs`. I think this test should pass without this change.